desktop/backend/python/controller/controls/mouse.py

```
@@ -0,0 +1,204 @@
+import pyautogui
+import time
+from typing import List, Tuple, Union
```

Ruff (0.14.10)

`typing.List` is deprecated, use `list` instead

(UP035)